app/src/views/SingleTeamView.vue

```diff
@@ -86,11 +87,27 @@
           />
         </template>
         <template #tab-2>
-          <div>transactions</div>
+          <BankTransactions
+            v-if="activeTab == 2"
+            :deposit-events="depositEvents as EventResult[]"
+            :depositEventLoading="depositEventLoading"
+            @get-deposit-events="getDepositEvents(team.bankAddress, BankEventType.Deposit)"
+            :transfer-events="transferEvents as EventResult[]"
+            :transferEventLoading="transferEventLoading"
+            @get-transfer-events="getTransferEvents(team.bankAddress, BankEventType.Transfer)"
+            :tips-address-changed-events="tipsAddressChangedEvents as EventResult[]"
+            :tips-address-changed-event-loading="tipsAddressChangedEventLoading"
+            @get-tips-address-changed-events="
+              getTipsAddressChangedEvents(team.bankAddress, BankEventType.TipsAddressChanged)
+            "
+            :send-to-wallet-events="sendToWalletEvents as EventResult[]"
+            :send-to-wallet-event-loading="sendToWalletEventLoading"
+            @get-send-to-wallet-events="
+              getSendToWalletEvents(team.bankAddress, BankEventType.PushTip)
+            "
+          />
```

Comment:
@farrellh1 the idea behind panel here is to move login inside some's component and regroup them.
So instead of sending element inside this component through props. Can you use them directly inside.

Reply:
oh the logic?